app/controllers/application_controller.rb

```diff
@@ -1,2 +1,3 @@
 class ApplicationController < ActionController::Base
+  include SessionsHelper
```

Comment:
helper dùng cho view thôi nha em, muốn viết fuction chung ở model hoăc controller thì dùng concern nha
tham khảo: https://dev.to/software_writer/how-rails-concerns-work-and-how-to-use-them-gi6

Reply:
Dạ để em xem